src/components/pages/admin/events/dashboard/orders/order/OrderItems.js

```diff
@@ -383,9 +385,10 @@ class OrderItems extends Component {
 		const {
 			name: eventName,
 			venue,
-			displayDate: eventDisplayDate
+			displayDate
 		} = eventDetails;
 
+		const eventDisplayDate = moment.tz(timezone || null).format("YYYY/DD/MM HH:mm A z");
```

Comment:
eventDetails.displayDate is already a formatted moment result. I don't know where else we use that `.format("llll")` so if we want to have our own formatting here we need to replace this line with something like:
```suggestion
		const eventDisplayDate = moment.utc(eventDetails.event_start).tz(timezone).format("YYYY/DD/MM HH:mm A z");
```
I have also removed the `|| null` because `timezone` is a required field.

Reply:
Thanks @Krakaw I have updated it and found the other area where it is being used and passed in the timezone and format.